app/helpers/client_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ClientHelper
+  def means_tested?(legal_aid_application)
```

Comment:
I would expect `method?` to return a boolean, or yes/no possibly. perhaps `means_tested_answer`, would be less confusing for our future selves

Reply:
Cool, thanks Joel, good point.  Originally it _was_ a boolean response, then I refactored it! 🤦‍♂️ 
I'll re-work and re-submit before moving into UAT 👍

Reply:
Updated to means_tested_description